tests/parsing/elixir/module.ex

```diff
@@ -1,8 +1,4 @@
-defmodule MyAppWeb.UserController do
-  use MyAppWeb, :controller
-
-  def show(conn, %{"id" => id}) do
-    user = Repo.get(User, id)
-    render(conn, :show, user: user)
-  end
+defmodule Number do
```

Comment:
These tests are just checking that the code can be parsed, so it doesn't really test the elixir-to-generic conversion, right?

Reply:
I guess we don't have that kind of test suite yet (we could write such tests using something like Cram perhaps). But I'll add some taint maturity tests in a subsequent PR.

Reply:
correct, at some point we should have tainting test or matching tests that exercise those code paths.